template/menu.html

```diff
@@ -3,7 +3,9 @@
     {{#user}}
       <h3 class="panel-title">Hello {{user.display_name}}</h3>
       <form method='post'>
-        <button type="submit" name='action' value='logout' class="btn btn-default">Log out</button>
+				{{> csrf_token}}
+				<button type="submit" name='action' value='logout' class="btn btn-default">Log out</button>
+				{{> csrf_token}}

```

Comment:
Two CSRF tokens?

Reply:
its twice as safe that way

But actually I accidentally added one before I added it to all the forms.